actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -159,6 +159,14 @@ def self.driven_by(driver, using: :chrome, screen_size: [1400, 1400], options: {
       self.driver = SystemTesting::Driver.new(driver, **driver_options, &capabilities)
     end
 
+    # Configuration for the System Test application server.
+    #
+    # By default this is localhost. This method allows the host and port to be specified manually.
+    def self.served_by(host:, port:)
+      Capybara.server_host = host
+      Capybara.server_port = port
```

Comment:
Shouldn't we also configure `Capybara.app_host` (so the Selenium container would be able to access the Rails test server)? 

See also https://evilmartians.com/chronicles/system-of-a-test-setting-up-end-to-end-rails-testing#dockerizing-system-tests

Reply:
Is it needed? The tests run with this configuration.

Reply:
> The tests run with this configuration.

yeah, that's probably due to the use of the shared network and the exposed test server port (so, `localhost:45678` works from the chrome service).